client/src/app/site/motions/modules/category/components/category-list/category-list.component.html

```diff
@@ -12,10 +12,37 @@ <h2 translate>Categories</h2>
     </div>
 </os-head-bar>
 
-<!-- Creating a new category -->
-<mat-card class="os-card" *ngIf="isCreatingNewCategory">
-    <mat-card-title>New category</mat-card-title>
-    <mat-card-content>
+<os-list-view-table
+    [repo]="repo"
+    [allowProjector]="false"
+    [columns]="tableColumnDefinition"
+    (dataSourceChange)="onDataSourceChange($event)"
+>
+    <div *pblNgridCellDef="'title'; row as category; rowContext as rowContext" class="cell-slot fill">
+        <a
+            class="detail-link"
```

Comment:
use an `onClick` handler and the `rowContext` to save the scroll-key
After navigation to the list view, the user is supposed to automatically scroll back to the last scroll position.